source/simulator/core.cc

```diff
@@ -2313,6 +2321,15 @@ namespace aspect
     // we disable automatic summary printing so that it won't happen when
     // throwing an exception. Therefore, we have to do this manually here:
     computing_timer.print_summary ();
+
+    pcout << "******************************************************************" << std::endl
+          << "Please see the following document for info on how to cite ASPECT:" << std::endl
```

Comment:
Instead of copying it twice, maybe have a function that provides the whole message as a string (`get_citation_information`?)